pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeforwardingrule/globalcomputeforwardingrule/_http.log

```diff
@@ -1041,7 +1041,7 @@ X-Xss-Protection: 0
   "insertTime": "2024-04-01T12:34:56.123456Z",
   "kind": "compute#operation",
   "name": "${operationID}",
-  "operationType": "SetTarget",
```

Comment:
Does the real GCP use `setTarget` on this field?

Reply:
Yes, so I updated the mock service and re-collected the log. This won't affect anything, I just wanted to ensure that the log remains as identical as possible.